pkg/detectors/speechtextai/speechtextai.go

```diff
@@ -0,0 +1,77 @@
+package speechtextai
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct{}
+
+// Ensure the Scanner satisfies the interface at compile time
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	//Make sure that your group is surrounded in boundry characters such as below to reduce false positives
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"speechtextai"}) + `\b([0-9a-f]{32})\b`)
```

Comment:
Can you update the keywords here and L30 to `speechtext`? The domain is `speechtext.ai` so it may be unlikely to find the string `speechtextai` nearby.